Shared/ConditionalStrikethroughModifier.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  ConditionalStrikethroughModifier.swift
+//  Simplists
+//
+//  Created by Tom Hartnett on 10/2/22.
+//
+
+import SwiftUI
+
+struct ConditionalStrikethroughModifier: ViewModifier {
+    var isComplete: Bool
+
+    func body(content: Content) -> some View {
+        if isComplete {
+            content
+                .foregroundColor(Color("TextSecondary"))
+                .modifier(IfAvailableStrikethroughModifier())
+        } else {
+            content
+        }
+    }
+}
+
+private struct IfAvailableStrikethroughModifier: ViewModifier {
+    func body(content: Content) -> some View {
+        if #available(iOS 16, *) {
```

Comment:
A downside of removing `FocusableTextField`:  strikethrough only available on iOS 16.